hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -384,6 +400,7 @@ private synchronized void flushWrittenBytesToServiceInternal(final long offset,
    * operation FIFO queue.
    */
   private synchronized void shrinkWriteOperationQueue() throws IOException {
+    outputStreamStatistics.queueShrunk();
```

Comment:
actually, queue is not shrunk here. Rather it is inside the while loop. will change this.